corehq/apps/analytics/tasks.py

```diff
@@ -38,6 +38,19 @@
 HUBSPOT_EXISTING_USER_INVITE_FORM = "7533717e-3095-4072-85ff-96b139bcb147"
 HUBSPOT_COOKIE = 'hubspotutk'
 
+ANALYTICS_RETRIES = 3
+ANALYTICS_SLEEP = 5
+
+def _persistent_analytics_post(func, retries=ANALYTICS_RETRIES, sleep=ANALYTICS_SLEEP):
+    for i in range(retries):
+        try:
+            return func()
+        except (requests.exceptions.HTTPError, requests.exceptions.BaseHTTPError):
+            if i < retries - 1:
+                time.sleep(sleep)
```

Comment:
OK, that's good. Might be worth adding a docstring with a warning that this could block for up to 15s.